HelpSource/Classes/Buffer.schelp

```diff
@@ -337,7 +337,7 @@ b.free;
 ::
 
 method:: loadDialog
-As link::#*read:: above, but gives you a load dialog window to browse for a file. Cocoa and SwingOSC compatible.
+As link::#*read:: above, but gives you a load dialog window to browse for a file. Cocoa compatible.
```

Comment:
I won't mark them all, but I think references to the old Cocoa GUI should be removed too. It's all Qt now.

Reply:
Removing those seem to be quite a big task - e.g. what is about

https://doc.sccode.org/Classes/SCDragView.html

Its even mentioned in build dependencies - maybe there would be some discussion needed if those things can really get deleted?

Reply:
> The Cocoa references should be removed in a separate pull request. One could deprecate them by adding a *new method to each: ... But perhaps that is too much trouble? I am not sure – what do you think jamshark70 ?

It appears that SCWindow etc all live only in the iPhone platform folder. They won't load on any desktop system. So I don't think it's necessary to deprecate their constructors.

A quick github search for SCWindow shows, however, that the help system still contains examples such as in Routine.schelp... not many but 😮  :

```
code::
// Using Routine to set button states on the fly.
(
var update, w, b;
w = SCWindow.new("State Window", Rect(150,SCWindow.screenBounds.height-140,380,60));

// a convenient way to set the button label
update = {
	|but, string| but.states = [[string.asString, Color.black, Color.red]];
	but.refresh;
};

b = SCButton(w, Rect(10,10,360,40));
b.font_(Font("Impact", 24));
```

But I agree that this can be a separate PR.